tests/containers/podman_image.pm

```diff
@@ -28,16 +28,15 @@ sub run {
     allow_selected_insecure_registries(runtime => $runtime);
 
     # We may test either one specific image VERSION or comma-separated CONTAINER_IMAGES
-    my $versions = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
-
-    my $dockerfile = ($host_distri ne 'suse') ? 'Dockerfile.python3' : 'Dockerfile';
+    my $versions   = get_var('CONTAINER_IMAGE_VERSIONS', get_required_var('VERSION'));
+    my $dockerfile = !check_os_release('suse', 'PRETTY_NAME') ? 'Dockerfile.python3' : 'Dockerfile';
     for my $version (split(/,/, $versions)) {
         my ($untested_images, $released_images) = get_suse_container_urls($version);
         my $images_to_test = check_var('CONTAINERS_UNTESTED_IMAGES', '1') ? $untested_images : $released_images;
         for my $iname (@{$images_to_test}) {
             record_info "IMAGE", "Testing image: $iname";
-            test_container_image(image => $iname, runtime => $runtime);
-            test_rpm_db_backend(image => $iname, runtime => $runtime);
+            #test_container_image(image => $iname, runtime => $runtime);
+            #test_rpm_db_backend(image => $iname, runtime => $runtime);
```

Comment:
Why this?

Reply:
stupid me man. i had done that for testing local and i forgot it. thats why

Reply:
**this change is ready for review**